### ISSUE TITLE
WebView doesn't show when app in background

### ISSUE THREAD

**Issue by onetech-dev** [gree/unity-webview#989]
Hello @KojiNakamaru 

First of all, thank you for the great plugin

I'm having trouble showing the webview, I press Home to close app in background then go back and the webview doesn't show up. Has anyone encountered this situation and posted an issue yet?

Thank you!

**Comment by KojiNakamaru** [comment]
The same issue was discussed in #979 and I introduced countermeasures in #983 and #985. Could you please try the latest binaries?

**Comment by onetech-dev** [comment]
Hi @KojiNakamaru 

Do you mean I have to clone source and update package?

![image](https://github.com/gree/unity-webview/assets/45479617/3602cab2-fce2-49c0-b977-ed48366ccf6c)

**Comment by KojiNakamaru** [comment]
You can download the latest from https://github.com/gree/unity-webview/blob/master/dist/unity-webview.zip .

**Comment by onetech-dev** [comment]
Thank @KojiNakamaru 

I already have the folders in Unity when I import package. 
Do I just need to replace these folders in Unity?

![image](https://github.com/gree/unity-webview/assets/45479617/264b7dee-3c24-4b59-b282-387cd78ddb32)

**Comment by KojiNakamaru** [comment]
Yes, please replace them by extracting the latest https://github.com/gree/unity-webview/blob/master/dist/unity-webview.zip .

**Comment by onetech-dev** [comment]
Thanks @KojiNakamaru 

It worked for me!

**Comment by G-r-i-n-c-h** [comment]
Hey @KojiNakamaru :) Had same problem with Unity 2021.x. After app was going to background and coming back webview was not displayed. have removed 
#if UNITY_2022_1_OR_NEWER
        webView.SetStatic<bool>("forceBringToFront", true);
#endif
From WebViewObject.cs and now its working fro 2021 too :) 
Posting so it can help other devs too :)

**Comment by KojiNakamaru** [comment]
@G-r-i-n-c-h Thanks! I'll later update the code.

**Comment by G-r-i-n-c-h** [comment]
> @G-r-i-n-c-h Thanks! I'll later update the code.

Could we consider introducing a dedicated function for invoking 'layout.bringToFront()' or possibly incorporate code within the 'SetVisibility' function? I'm uncertain of the cause, but it appears there may be an issue with Unity, particularly with newer versions, where occasionally layout become hidden without going in background. Having this capability could greatly assist others in resolving similar issues independently.

**Comment by KojiNakamaru** [comment]
@G-r-i-n-c-h Thank you for the suggestion. I modified SetVisibility in #1000 and updated binaries.

**Comment by onetech-dev** [comment]
Hello @KojiNakamaru 

I'm facing this problem:
I have two scenes Main and Webview.

1. Main => Webview => Webview is show
2. Back to Main scene
3. Turn off phone screen and turn on again
4. Run step 1 again: Main => Webview => Webview not show

It happens on Android devices (Tested on Samsung Galaxy S22)
iPhone is fine.

**Comment by KojiNakamaru** [comment]
@onetech-dev 
Could you please provide a minimum project that reproduces the issue, based on the sample app?

**Comment by onetech-dev** [comment]
> The same issue was discussed in #979 and I introduced countermeasures in #983 and #985. Could you please try the latest binaries?

Hi @KojiNakamaru 

Sorry, my problem is the same as the one I quoted above. Updated to the latest binaries and now works perfectly 👍
Thank you so much!
